desafio1.html

```diff
@@ -21,32 +21,52 @@
 <body>
     <h1>Game Reviews</h1>
     <h2>Crie um novo review</h2>
-    <section>Título do Game</section>
-    <input type="text">
-    <section>Review do Jogo</section>
-    <textarea></textarea> 
-    <form>
-        Categoria do Jogo
-            <select name="categoriajogo">
+    <form id="new_review_form" >
+    <label for="title">Título do Game</label>
```

Comment:
O que antes estava como tag `<section>`, alterei apra `<label>`.
A tag `<label>` ela é usada para criar **nome/titulo/rótulo** para o input. Além de trazer a semântica correta pra tag, a tag label possui algumas functionalidades. Como podes bem observar, o tributo `**for**` dentro da tag label, conecta a label ao input desejado.
Dessa forma você tem um comportamento nativo do HTML de ao clicar com o mouse na label, o input fica focado, pronto para digitar o texto.
Vale ressaltar que o `**for**` tem que ter o mesmo valor do `**id**` do **input** que vc quer ligar. Nesse caso a tag label com o `for="title"` se conecta com o input com `id="title"` tbm.


A tag <section> nao é uma tag de texto. Ela deve ser usada para criar seções de conteúdo na sua página.

Você poderia englobar todo o formulário em uma tag section. Isso faria sentido pois haverá uma seção na sua página destinada à criação. Posteriormente quando você for fazer a listagem dos dados criados nesse formulário, essa seção de listagem pode ficar englobada em outra tag section.
Fazendo dessa forma vc cria a semantica correta para ter uma seção destinada a criação e outra para a listagem. 
Isso é importante apenas para semântica. Tags <section> nao criam elementos visuais na página